content/02.main-text.md

```diff
@@ -120,14 +125,20 @@ Alternatively, a reviewer can post feedback with a pseudonymous GitHub account o
 Authors can elect to respond to reviews in the GitHub issues or a public [response letter](https://github.com/greenelab/meta-review/blob/813f461fe719d8fe7bcda78871cfad106f83dfe7/content/response-to-reviewers.md), creating open peer review.
 ** TODO: update `response-to-reviewers.md` URL once it is finalized **
 
+Although we developed Manubot with collaborative writing in mind, it can also be helpful for individuals preparing scholarly documents.
+Authors may choose to make their changes directly to the `master` branch, forgoing pull requests and reviews.
+This workflow retains many of Manubot's benefits, such as transparent history, automation, and allowing outside contributors to propose changes.
+In cases where outside contributions are unwanted, authors can disable pull requests on GitHub.
+See the [existing manuscripts](#existing-manuscripts) for examples of the range of contribution workflows and Manubot use cases.
```

Comment:
`(#existing-manuscripts)` can be removed and this should still work. See [Pandoc's](https://pandoc.org/MANUAL.html#heading-identifiers) `implicit_header_references`